content/en/docs/ops/diagnostic-tools/cni/index.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Troubleshooting CNI
```

Comment:
small architecture diagram may help understand users how this works?

Reply:
You mean architecture for overall how istio cni works, or just log part?

Reply:
Sounds reasonable to put on main page. I generally meant overall, but it could include the logging portion. I think most people don't actually understand CNI, especially that the daemonset doesn't really do the iptables injection itself, etc

Reply:
Done.